cli/lib/kontena/cli/master/login_command.rb

```diff
@@ -117,7 +117,12 @@ def set_server_token(server)
     def use_authorization_code(server, code)
       vspinner "Exchanging authorization code for an access token from Kontena Master" do
         client = Kontena::Client.new(server.url, server.token)
-        response = client.exchange_code(code) rescue nil
+        begin
+          response = client.exchange_code(code)
+        rescue
+          ENV["DEBUG"] && puts("#{$!} : #{$!.message}\n#{$!.backtrace.join("  \n")}")
+          exit_with_error "Code exchange failed: #{$!.message}"
```

Comment:
Alternatively, raise an exception and let the top-level DEBUG/error code handle the formatting?